pkg/app/cmd.go

```diff
@@ -211,6 +211,7 @@ func (o *options) bindFlags(flags *pflag.FlagSet) {
 		defaultRunID = uuid.New().String()
 	}
 	flags.StringVar(&o.runid, "run-id", defaultRunID, "unique identifier for a kubetest2 run")
+	flags.BoolVar(&o.uploadRundir, "upload-rundir", false, `To decide whether the test binaries and run specific metadata have to be uploaded to the ARTIFACTS `)
```

Comment:
I think we should rename this, `kubetest2` does not "upload" anything itself.

Also: perhaps we should have an option to simply have rundir be under artifacts again? I'm not sure what that should look like without thinking about it some more ...

Reply:
@BenTheElder How about renaming it to `cpRundirToArtifacts`? (Please suggest any better name for the flag)

Also: Yes I did the same. When the flag `uploadRundir` was set to true, I was copying artifacts.RunDir() to artifacts.BaseDir()
https://github.com/kubernetes-sigs/kubetest2/blob/49251e4560b1c5c51eccb0458d0f8a83ea880763/pkg/app/app.go#L177